plugins/browser/frontend/public/stylesheets/browser.scss

```diff
@@ -50,39 +50,32 @@
         } 
     }
 
-    h4 {
-        font-family: Inter;
-        font-style: normal;
-        font-weight: 500;
-        font-size: 16px;
-        line-height: 24px;
-        color:#333C48;
-    }
+ 
 
-    h5 {
-        font-family: Inter;
-        font-style: normal;
-        font-weight: 500;
-        font-size: 14px;
-        line-height: 24px;
-        color: #333C48;
-        margin-bottom:20px;
+    .cly-vue-metric-card {
```

Comment:
Would you think that it is possible to make this generic? If it is so we should better move it to core styling, rather than keeping it in a plugin :slightly_smiling_face:

Reply:
I removed these lines. I used "bulma" for some margin and background properties 🙂